graph/appender/unused_node_test.go

```diff
@@ -1,9 +1,10 @@
 package appender
 
 import (
-	"github.com/kiali/kiali/models"
 	"testing"
 
+	"github.com/kiali/kiali/models"
+
 	"github.com/kiali/kiali/config"
 	"github.com/kiali/kiali/graph"
 	"github.com/stretchr/testify/assert"
```

Comment:
this is a third-party, non-kiali, import. It should be moved up above the kiali group of imports into its own group to match our coding standards.

go-imports
blank-line
third-party imports
blank-line
kiali imports